src/app/components/OfflineBanner/index.tsx

```diff
@@ -26,12 +27,19 @@ export const NetworkOfflineBanner: FC<{ wantedNetwork?: Network }> = ({ wantedNe
   const scope = useScopeParam()
   const { t } = useTranslation()
   const targetNetwork = wantedNetwork || scope?.network || Network.mainnet
-  const isNetworkOffline = useIsApiOffline(targetNetwork)
+  const isNetworkReachable = useIsApiReachable(targetNetwork)
   const networkNames = getNetworkNames(t)
   const target = networkNames[targetNetwork]
-  return isNetworkOffline ? (
-    <StyledAlert severity="warning">{t('home.apiOffline', { target })}</StyledAlert>
-  ) : null
+  if (!isNetworkReachable.reachable) {
+    if (isNetworkReachable.reason === 'userOffline') {
```

Comment:
I would prefer adding the full text of reason right inside the hook (we have access to translation there, too); no need to have a layer of compilation here...